test/fw/ptl/utils/pbs_procutils.py

```diff
@@ -235,20 +241,26 @@ def __init__(self, name=None, pid=None):
         self.rss = None
         self.vsz = None
         self.pcpu = None
+        self.pmem = None
+        self.size = None
+        self.cputime = None
         self.time = time.time()
         self.command = None
 
     def __str__(self):
-        return "%s pid: %s rss: %s vsz: %s pcpu: %s command: %s" % \
+        return "%s pid: %s rss: %s vsz: %s pcpu: %s pmem: %s \
```

Comment:
You can replace this with an fstring:
```Python
return f"{self.name} pid: {self.pid} rss: {self.rss} ..."
```
I find it more readable than % formatting, and it's faster. You also don't need to call str() on the values.

Reply:
@modified